pkgs/by-name/in/invidtui/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, yt-dlp, ffmpeg, mpv }:
+
+buildGoModule rec {
+  pname = "invidtui";
+  version = "0.3.6";
+
+  src = fetchFromGitHub {
+    owner = "darkhz";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-zUr0zrIJPpqhHvL7PFFN7cgcgBXV+WHO/eRes7+HzxM=";
+  };
+
+  vendorHash = "sha256-cKvY3/3N3SESBVol7Af3M3mJaPwxLzd/rKN8P+qh7sY=";
+
+  doCheck = false;
```

Comment:
Why are the checks disabled? It seems to build with checks enabled, too.

Reply:
You're right. I initially disabled checks and simply forgot that I could re-enable them. I enabled them now.